__tests__/integration/one-app.spec.js

```diff
@@ -876,8 +879,8 @@ describe('Tests that can run against either local Docker setup or remote One App
           }
         );
         expect(response.status).toBe(200);
-        expect(response.headers._headers).toHaveProperty('access-control-allow-origin');
-        expect(response.headers._headers['access-control-allow-origin']).toEqual(['test.example.com']);
+        expect(response.headers.raw()).toHaveProperty('access-control-allow-origin');
```

Comment:
Would this new API of `raw` affect other wrappers or tools?

Reply:
Should not, the versions of `node-fetch` and `whatwg-fetch` are much closer aligned. the `headers._headers` is a node-fetch@v1 only api.